<ui version="4.0" >
 <class>fifteenPuzzleConfig</class>
 <widget class="QWidget" name="fifteenPuzzleConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configure Fifteen Puzzle</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QGroupBox" name="gb_appearance" >
     <property name="title" >
      <string>Appearance</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QRadioButton" name="rb_identical" >
        <property name="text" >
         <string>Use plain pieces</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QRadioButton" name="rb_split" >
          <property name="text" >
           <string>Use custom image:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KUrlRequester" name="urlRequester" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_showNumerals" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Show numerals</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_actions" >
     <property name="title" >
      <string>Actions</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QPushButton" name="pb_shuffle" >
        <property name="text" >
         <string>Shuffle Pieces</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rb_split</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_showNumerals</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>146</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel" >
     <x>146</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_split</sender>
   <signal>toggled(bool)</signal>
   <receiver>urlRequester</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>146</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel" >
     <x>171</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
